app/ml.py

```diff
@@ -492,4 +492,43 @@ async def get_weather_conditions(city: City):
     city = validate_city(city)
     weather_cond = await select_weather_conditions(city)
     
-    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
\ No newline at end of file
+    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
```

Comment:
I changed weather_temperature back to the original which was weather_conditions. I didn't see any reason for the change plus it was going to mess up the function here. I'm assuming a typo?

Reply:
Julio, nice catch!  It is definitely a typo. Good job fixing it!

Reply:
You guys are amazing!